src/SCREENS.ts

```diff
@@ -30,14 +49,162 @@ const SCREENS = {
+        SIGN_IN: 'SignIn',
+        PRIVATE_NOTES: 'Private_Notes',
+        ROOM_MEMBERS: 'RoomMembers',
+        ROOM_INVITE: 'RoomInvite',
+        REFERRAL: 'Referral',
+    },
     SIGN_IN_WITH_APPLE_DESKTOP: 'AppleSignInDesktop',
     SIGN_IN_WITH_GOOGLE_DESKTOP: 'GoogleSignInDesktop',
     DESKTOP_SIGN_IN_REDIRECT: 'DesktopSignInRedirect',
     SAML_SIGN_IN: 'SAMLSignIn',
+
+    MONEY_REQUEST: {
+        ROOT: 'Money_Request',
+        AMOUNT: 'Money_Request_Amount',
+        PARTICIPANTS: 'Money_Request_Participants',
+        CONFIRMATION: 'Money_Request_Confirmation',
+        CURRENCY: 'Money_Request_Currency',
+        DATE: 'Money_Request_Date',
+        DESCRIPTION: 'Money_Request_Description',
+        CATEGORY: 'Money_Request_Category',
+        TAG: 'Money_Request_Tag',
+        MERCHANT: 'Money_Request_Merchant',
+        WAYPOINT: 'Money_Request_Waypoint',
+        EDIT_WAYPOINT: 'Money_Request_Edit_Waypoint',
+        DISTANCE: 'Money_Request_Distance',
+        RECEIPT: 'Money_Request_Receipt',
+    },
+
+    IOU_SEND: {
+        ADD_BANK_ACCOUNT: 'IOU_Send_Add_Bank_Account',
+        ADD_DEBIT_CARD: 'IOU_Send_Add_Debit_Card',
+        ENABLE_PAYMENTS: 'IOU_Send_Enable_Payments',
+    },
+
+    REPORT_SETTINGS: {
+        ROOT: 'Report_Settings_Root',
+        ROOM_NAME: 'Report_Settings_Room_Name',
+        NOTIFICATION_PREFERENCES: 'Report_Settings_Notification_Preferences',
+        WRITE_CAPABILITY: 'Report_Settings_Write_Capability',
+    },
+
+    NEW_TASK: {
+        ROOT: 'NewTask_Root',
+        TASK_ASSIGNEE_SELECTOR: 'NewTask_TaskAssigneeSelector',
+        TASK_SHARE_DESTINATION_SELECTOR: 'NewTask_TaskShareDestinationSelector',
+        DETAILS: 'NewTask_Details',
+        TITLE: 'NewTask_Title',
+        DESCRIPTION: 'NewTask_Description',
+    },
+
+    TASK: {
+        TITLE: 'Task_Title',
+        DESCRIPTION: 'Task_Description',
+        ASSIGNEE: 'Task_Assignee',
+    },
+
+    PRIVATE_NOTES: {
+        VIEW: 'PrivateNotes_View',
+        LIST: 'PrivateNotes_List',
+        EDIT: 'PrivateNotes_Edit',
+    },
+
+    REPORT_DETAILS: {
+        ROOT: 'Report_Details_Root',
+        SHARE_CODE: 'Report_Details_Share_Code',
+    },
+
+    WORKSPACE: {
+        INITIAL: 'Workspace_Initial',
+        SETTINGS: 'Workspace_Settings',
+        CARD: 'Workspace_Card',
+        REIMBURSE: 'Workspace_Reimburse',
+        RATE_AND_UNIT: 'Workspace_RateAndUnit',
+        BILLS: 'Workspace_Bills',
+        INVOICES: 'Workspace_Invoices',
+        TRAVEL: 'Workspace_Travel',
+        MEMBERS: 'Workspace_Members',
+        INVITE: 'Workspace_Invite',
+        INVITE_MESSAGE: 'Workspace_Invite_Message',
+        CURRENCY: 'Workspace_Settings_Currency',
+    },
+
+    EDIT_REQUEST: {
+        ROOT: 'EditRequest_Root',
+        CURRENCY: 'EditRequest_Currency',
+    },
+
+    I_KNOW_A_TEACHER: 'I_Know_A_Teacher',
+    INTRO_SCHOOL_PRINCIPAL: 'Intro_School_Principal',
+    I_AM_A_TEACHER: 'I_Am_A_Teacher',
```

Comment:
Should these be solid?
```suggestion
    I_KNOW_A_TEACHER: 'IKnowATeacher',
    INTRO_SCHOOL_PRINCIPAL: 'IntroSchoolPrincipal',
    I_AM_A_TEACHER: 'IAmATeacher',
```